tasks/src/test/java/taf/test/WebDriverYandexDiskRecycleBinTest.java

```diff
@@ -0,0 +1,17 @@
+package taf.test;
+
+import org.testng.Assert;
+import org.testng.annotations.*;
+import taf.service.YandexDiskService;
+
+public class WebDriverYandexDiskRecycleBinTest extends CommonConditions{
+
+  @Test(description = "Check trash is empty")
+  public void verifyTrashIsEmpty() {
+    if (!new YandexDiskService(driver).trashIsEmpty()) {
```

Comment:
Why do you need the block of code? It looks strange
What is the difference between trashIsEmpty and isTrashEmpty methods?